Bos Taurus paragraph and other edits

gregorgorjanc opened the pull request:
Here is the Bos Taurus paragraph and a number of other edits - hopefully an improvement ;)

gregorgorjanc committed:
Intro edits

gregorgorjanc committed:
Edits for Tutorial and parts of Application

gregorgorjanc committed:
Bos Taurus and more edigs

gregorgorjanc committed:
Clarifying that we need directional or stabilising selection on multiple loci across chromosomes to get between chromosome correlations

lauterbur commented on adding_species_manuscript.md:
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
-
-        -   can be used for methods development and inference
+    
+We have added Bos Taurus (cattle) species as an example of an agricultural
```
_Bos taurus_

lauterbur commented on adding_species_manuscript.md:
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
+We have added Bos Taurus (cattle) species as an example of an agricultural
+species. Agricultural species have generally experienced strong reduction in
+effective population size over time due to selection caused by domestication and
+recently selective breeding. Both of these processes have occurred over a
```
~~recently~~ selective breeding

lauterbur commented on adding_species_manuscript.md:
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
+breeding (e.g., @VanRaden2020, @Makanjouloa2020). This decrease is interesting
+because it is challenging the estimation of demography (@MacLeod2013),
+genome-wide association and prediction (@MacLeod2014), and inference of selection
+(@Hartfield2022). With respect to the parameters chosen in the stdpopsim
```
I think we can consolidate the introductory paragraph here: Suggest replacing lines 343-355 with "In cattle, high quality genome assemblies are now available for several breeds and population genomic analyses have become widely used to improve selective breeding and genomic prediction. Their small effective population size (~90 around 1980, and continuing to decline due to intense selective breeding) makes demographic and selection inference, as well as genome-wide association and prediction, challenging. For these reasons, it was useful to develop a standardized population genomic cattle model for stdpopsim."

lauterbur replied:
(with appropriate citations of course)

lauterbur commented on adding_species_manuscript.md:
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
+breeding (e.g., @VanRaden2020, @Makanjouloa2020). This decrease is interesting
+because it is challenging the estimation of demography (@MacLeod2013),
+genome-wide association and prediction (@MacLeod2014), and inference of selection
+(@Hartfield2022). With respect to the parameters chosen in the stdpopsim
```
New paragraph at "With respect to the parameters..."

gregorgorjanc committed:
Minor tweaks in cattle paragraph

lauterbur commented:
Unless you want to change anything before I do, I will go ahead and merge these so I can do all-over edits.

gregorgorjanc committed:
Addressing suggestions from @lauterbur

@lauterbur do you suggest I cut the 338 "We have added Bos taurus (cattle) as an ..." even more?

lauterbur commented:
I don't think Bos taurus needs any more cuts/major edits for now. We'll see how things flow with the rest once we have the conclusion.
Thanks for the edits elsewhere too, very helpful!

gregorgorjanc commented:
@lauterbur yeah merge this in! Thanks! Sorry for dragging my feet!

One comment though - we need something similar for all other added species too, right?

lauterbur commented:
I think that's overkill, one example for why a species might be of general interest, and how to pick parameter values, is probably enough since this is now a more general "how to do a popgen simulation" paper rather than a "uses of stdpopsim" paper

gregorgorjanc commented:
OK, I was thinking the same, just wasn’t sure if the direction of travel.

With regards,
  Gregor

On Mon, Apr 11 2022 at 7:03 pm, M. Elise Lauterbur ***@***.***>
wrote:

> I think that's overkill, one example for why a species might be of general
> interest, and how to pick parameter values, is probably enough since this
> is now a more general "how to do a popgen simulation" paper rather than a
> "uses of stdpopsim" paper
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/popsim-consortium/adding-species-manuscript/pull/25#issuecomment-1095365477>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFRGSXNWITTRGTRKKW3WT3VERSPBANCNFSM5TBWJEAQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

gregorgorjanc committed:
Update for the mutation rate (done also in the code!)
